src/js/actions/edit.js

```diff
@@ -422,7 +422,7 @@ define(function (require, exports) {
     };
     redo.action = {
         reads: [locks.JS_APP, locks.JS_DOC],
-        writes: [locks.JS_UI],
+        writes: [locks.JS_PANEL],
```

Comment:
this seems safe to me, @pineapplespatula ... since it is now consistent with `undo`.